Node/src/TeamsChatConnector.ts

```diff
@@ -110,6 +112,17 @@ export class TeamsChatConnector extends builder.ChatConnector {
   }
 
   /**
+  * This method is to resolve regex or LUIS Bot being affected by at mentions
+  * Set true would have incoming messages stripped off source Bot at mentions
+  * The default would be false
```

Comment:
Now that this is this is opt-in, I'm curious as to why we'd do it this way instead of middleware. IIRC the reason to do it in the connector was that the developer might forget to add the middleware, but doesn't that apply here too?